app/assets/javascripts/code/services/sessions_downloader.js

```diff
@@ -48,10 +48,18 @@ angular.module("aircasting").factory("sessionsDownloader", ['$rootScope', '$http
       }).value();
 
       session.$selected = sessionIds.include(session.id);
+
+      setDefaultSessionAttributes(session);
     });
     sessions.push.apply(sessions, data);
     sessions = orderBy(sessions, '$selected', 'end_time_local');
   };
 
+  const setDefaultSessionAttributes = function (session) {
```

Comment:
Have you considered using `session => {}`. Also, that could be private (out of the `sessions_downloader` object)